docs/installation.rst

```diff
@@ -34,6 +34,6 @@ Finally, perform an `--editable` install of figanos:
 
 .. code-block:: console
 
```

Comment:
Should we use python -m pip install or  flit install -s/--symlink?

Reply:
The main differences:

- `$ pip install .` → installs a non-development version of the library into your workspace (for general users).
- `$ flit install --symlink` (no `.`, will correct that) is the `flit` equivalent to `pip install --editable .` (`--editable` *does not* work as intended with `flit`). This creates a symlinked folder in your python libraries to the git clone.

The `$ python -m` is to prevent users in Windows/macOS/Linux from running `pip` with the wrong environment (i.e. a new environment in virtualenv/conda that does not have `pip` installed will use the *system*-installed `pip` instead; Mistakes have been made before).

Reply:
Should
Finally, perform an --editable install of figanos: `$ python -m pip install `
become
Finally, perform an editable install of figanos: `$ flit install --symlink`

in installation.rst or am I stil confused on which one to use?

Reply:
It would be the second command, my bad. Updating this now.